### PR TITLE
C# SDK for Azure Batch Management 7.0

### PR DESCRIPTION
**Pull request by matthchr**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR. The Swagger PR is [here](https://github.com/Azure/azure-rest-api-specs/pull/4720).
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by matthchr** [comment]
Please note: There _are_ breaking changes. That is why this is a breaking change release (6.0 -> 7.0).

Please help me confirm I did the right thing for this bullet item: `The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.` - I think that this happens automatically in the `src/SDKs/_metadata/batch_resource-manager.txt` file, but if there's more that I have to do on top of that please let me know where it is.

File: src/SDKs/Batch/Management/Management.Batch/Microsoft.Azure.Management.Batch.csproj [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
-		<AssemblyTitle>Microsoft Azure Batch Management Library</AssemblyTitle>
-		<AssemblyName>Microsoft.Azure.Management.Batch</AssemblyName>
-		<Version>6.0.0</Version>
-		<PackageTags>Microsoft Azure batch management;batch;</PackageTags>
-		<PackageReleaseNotes>For detailed release notes, see: https://aka.ms/batch-net-management-changelog</PackageReleaseNotes>
-	</PropertyGroup>
-	<PropertyGroup>
-		<TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
-	</PropertyGroup>
+  <PropertyGroup>
```
Comment (dsgouda) [inline src/SDKs/Batch/Management/Management.Batch/Microsoft.Azure.Management.Batch.csproj]: nit: please fix indentation

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please reference the REST spec API PR here

**Comment by matthchr** [comment]
The spec was already referenced in the original PR content.

> Please add REST spec PR link to the SDK PR. The Swagger PR is here.

File: src/SDKs/Batch/Management/Management.Batch/Microsoft.Azure.Management.Batch.csproj (hunk 1 above)
Comment (matthchr) [reply src/SDKs/Batch/Management/Management.Batch/Microsoft.Azure.Management.Batch.csproj]: This is fixed indentation - the indentation should be spaces, not tabs, right? It was tabs and I changed it to spaces.

**Review by dsgouda — APPROVED** [review]
LGTM

**Comment by dsgouda** [comment]
Will merge once CIs pass

**Pull request merged**